server/src/main/java/org/opensearch/index/shard/IndexShard.java

```diff
@@ -2996,8 +2996,9 @@ public ReplicationStats getReplicationStats() {
             final Set<SegmentReplicationShardStats> stats = getReplicationStatsForTrackedReplicas();
             long maxBytesBehind = stats.stream().mapToLong(SegmentReplicationShardStats::getBytesBehindCount).max().orElse(0L);
             long totalBytesBehind = stats.stream().mapToLong(SegmentReplicationShardStats::getBytesBehindCount).sum();
+            // currentReplicationLagMillis is replication lag that takes into account time taken to upload data to remote store
             long maxReplicationLag = stats.stream()
-                .mapToLong(SegmentReplicationShardStats::getCurrentReplicationTimeMillis)
+                .mapToLong(SegmentReplicationShardStats::getCurrentReplicationLagMillis)
```

Comment:
I think we should expose both the lags (end to end which includes the remote upload time) and the replication only lag.

Reply:
Lets pls take this up as a separate change to this fix.

Reply:
Tracking addition of new field here: Once name is fixed, can add it in: https://github.com/opensearch-project/OpenSearch/issues/10775